matchzoo/losses.py

```diff
@@ -0,0 +1,56 @@
+"""Self defined loss function."""
+import typing
+from keras import backend as K
+from keras import layers
+
+# TODO: Loss function parameters setting.
+margin = 1.0
+neg_num = 1
+
+
+def rank_hinge_loss(y_true: typing.Any,
+                    y_pred: typing.Any) -> typing.Any:
```

Comment:
`typing.Any` is not the best choice for the type of parameters. At least, you should list some possible types, such as `np.ndarray`.

Reply:
The inputs and outputs type of loss function is `Tensor`, which should be represented as `typing.Any`

Reply:
maybe `Typing.Iterable` is better?